frontend/src/component/common/NewConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEditBody/FreeTextInput/FreeTextInput.tsx

```diff
@@ -53,6 +55,13 @@ const useStyles = makeStyles()((theme) => ({
     valuesContainer: { marginTop: '1rem' },
 }));
 
+const LimitContainer = styled(Box)(({ theme }) => ({
+    backgroundColor: theme.palette.background.default,
+    '&:has(*)': {
+        marginTop: theme.spacing(2),
```

Comment:
this pattern will be propagated to other components in the next PR

Reply:
You could wrap the whole styling block here, but because it collapses completely when it's empty, it probably doesn't matter here 😄